app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if command.startswith("cp"):
+        command = command.replace("cp ", "")
+    else:
+        print("Unknown command")
+        return
+    list_of_files_name = command.split(" ")
+    if (list_of_files_name[0] != list_of_files_name[1]
+            and len(list_of_files_name) == 2):
```

Comment:
The condition should ensure that the command string contains exactly three elements: 'cp', source filename, and destination filename. Currently, it only checks for two elements, which is incorrect according to the task requirements.